RecoHI/HiTracking/python/HIBestVertexSequences_cff.py

```diff
@@ -48,4 +49,5 @@
     useFinalAdaptiveVertexCollection = cms.bool(True),
     finalAdaptiveVertexCollection = cms.InputTag("hiBestOfflinePrimaryVertex")
 )
-bestFinalHiVertex = cms.Sequence(hiOfflinePrimaryVertices * hiBestOfflinePrimaryVertex * hiSelectedVertex ) # vertexing run AFTER tracking
+bestFinalHiVertexTask = cms.Task(hiOfflinePrimaryVertices , hiBestOfflinePrimaryVertex , hiSelectedVertex )
+bestFinalHiVertex = cms.Sequence(bestFinalHiVertexTask) # vertexing run AFTER tracking
```

Comment:
This was only needed in RecoHI/HiTracking/python/HiTracking_cff.py: sequence not needed any more now, please remove